examples/2d/elixir_euler_unstructured_quad_periodic.jl

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Yeah, i needed the limiters such that I can use the fancy SSPRK43 method that did not require a `calc_time_step` callback. However, one of the `CurvedMesh` branches should have this time step functionality now. So this could get improved

Reply:
OK, sounds great!